# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleMapKey.h
# Commit: 7d5683180ac3
# Full Hash: 7d5683180ac3f9e7cc96d68dde27b8b48642a7ac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r a48aff5f2555 -r 7d5683180ac3 js/loader/ModuleMapKey.h
--- a/js/loader/ModuleMapKey.h	Thu Feb 17 15:57:52 2022 +0000
+++ b/js/loader/ModuleMapKey.h	Thu Feb 17 15:57:52 2022 +0000
@@ -8,7 +8,7 @@
 #include "nsURIHashKey.h"
 #include "nsIGlobalObject.h"
 
-namespace mozilla::dom {
+namespace JS::loader {
 
 class ModuleMapKey : public nsURIHashKey {
  public:
@@ -42,6 +42,6 @@
   ImplCycleCollectionTraverse(aCallback, aField.mWebExtGlobal, aName, aFlags);
 }
 
-}  // namespace mozilla::dom
+}  // namespace JS::loader
 
 #endif